OBAKit/Models/dao/OBAModelDAO.m

```diff
@@ -483,53 +501,8 @@ - (void)setVisited:(BOOL)visited forSituationWithId:(NSString*)situationId {
     [_preferencesDao writeVisistedSituationIds:_visitedSituationIds];
 }
 
-#pragma mark - Custom API Server
-
-- (void)addCustomApiUrl:(NSString *)customApiUrl {
-
-    if (!customApiUrl) {
-        return;
-    }
-
-    NSString *existingCustomApiUrl = nil;
-
-    for (NSString *recentCustomApiUrl in _mostRecentCustomApiUrls) {
-        if ([recentCustomApiUrl isEqualToString:customApiUrl]) {
-            existingCustomApiUrl = customApiUrl;
-            break;
-        }
-    }
-
-    if (existingCustomApiUrl) {
-        [_mostRecentCustomApiUrls removeObject:existingCustomApiUrl];
-        [_mostRecentCustomApiUrls insertObject:existingCustomApiUrl atIndex:0];
-    }
-    else {
-        [_mostRecentCustomApiUrls insertObject:customApiUrl atIndex:0];
-    }
-
-    NSInteger over = [_mostRecentCustomApiUrls count] - kMaxEntriesInMostRecentList;
-    for (NSInteger i=0; i<over; i++) {
-        [_mostRecentCustomApiUrls removeObjectAtIndex:_mostRecentCustomApiUrls.count - 1];
-    }
-
-    [_preferencesDao writeMostRecentCustomApiUrls:_mostRecentCustomApiUrls];
-}
-
 - (NSString*)normalizedAPIServerURL {
```

Comment:
check to see how easy it would be to replace this with something that would just return an NSURL